app/javascript/app/pages/regions/climate-sectoral-plan/climate-sectoral-plan-selectors.js

```diff
@@ -0,0 +1,50 @@
+import { createStructuredSelector, createSelector } from 'reselect';
+import { getTranslation } from 'utils/translations';
+import { getProvince } from 'selectors/provinces-selectors';
+import isEmpty from 'lodash/isEmpty';
+import { lowerDeburr } from 'utils/utils';
+
+export const getSectionsContent = ({ SectionsContent }) =>
+  SectionsContent && SectionsContent.data;
+
+const getQuery = ({ location }) => location && (location.query || null);
+
+const getClimateSectoralPlan = ({ climatePlans }) =>
+  climatePlans && climatePlans.data;
+
+const getSearch = createSelector(getQuery, query => {
+  if (!query || !query.search) return null;
+  return query.search;
+});
+
+export const getFilteredDataBySearch = createSelector(
```

Comment:
It's not necessary but maybe we could extract this selector as a kind of `util` selector. I saw the same thing in climate funding page. Something like:

```
function createTextSearchSelector(getData, getSearch) {
   return createSelector([getData, getSearch], (data, search) => {
   ...
   })
} 
```